paywall/src/unlockTypes.ts

```diff
@@ -67,9 +67,12 @@ export interface PaywallConfigLock {
   name: string
 }
 
+export type PaywallTypes = 'adblock' | 'paywall'
```

Comment:
Something like `KindOfPaywall` might be less ambiguous (`PaywallTypes` is a TS type whose allowed values are "types" of a different nature) but that's a matter of taste more than anything